pkg/kubelet/kubelet.go

```diff
@@ -682,7 +687,19 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 			utilfeature.DefaultFeatureGate.Enabled(features.PodAndContainerStatsFromCRI))
 	}
 
-	klet.pleg = pleg.NewGenericPLEG(klet.containerRuntime, plegChannelCapacity, plegRelistPeriod, klet.podCache, clock.RealClock{})
+	// adjust PLEG relisting period and threshold to higher value when Event PLEG is turned on
+	if utilfeature.DefaultFeatureGate.Enabled(features.EventPLEG) {
+		plegRelistPeriod = time.Second * 300
```

Comment:
let's  move these constants up next to `plegRelistPeriod` so we have all the constants in the right spot.